src/body.rs

```diff
@@ -257,7 +256,7 @@ impl Body {
     /// # Examples
     ///
     /// ```
-    /// # fn main() -> Result<(), http_types::Error> { async_std::task::block_on(async {
+    /// # fn main() -> Result<(), http_types::Error> { futures_lite::future::block_on(async {
```

Comment:
Can we keep `async_std` in the examples?

Reply:
fixed